src/adapters/watis.js

```diff
@@ -69,8 +70,40 @@ function findMatch(string) {
   }
   return parsed.replace(/\s+/g, ' ').trim();
 }
+async function addDescription(afkortingkje, description,event){
```

Comment:
eslint even bekijken, deze zorgt ervoor dat je file steeds deftig geformateerd is. In de root van het project staan een eslintrc.json die dit aanstuurt. Als je `npm run lint` uitvoert zie je nog waar er zaken aangepast zouden kunnen worden.

Reply:
Ik zal hier direct eens naar kijken